cerbero/bootstrap/linux.py

```diff
@@ -53,7 +53,9 @@ def start(self):
             if self.assume_yes:
               tool += ' ' + self.yes_arg;
             tool += ' ' + self.command;
+            self.config._restore_environment()
```

Comment:
A better commit message would be nice.

For instance adding the error.

Reply:
real issue is 

Without this commit, the following error happens in Centos 7, because
`yum install` is executed inside the CERBERO_PREFIX, and because
yum uses `python2` and `pycurl.so`, it ends up with undef symbols:

```
LD_LIBRARY_PATH="/opt/rh/devtoolset-7/root/usr/lib64:/opt/rh/devtoolset-7/root/usr/lib:/opt/rh/devtoolset-7/root/usr/lib64/dyninst:/opt/rh/devtoolset-7/root/usr/lib/dyninst:/opt/rh/devtoolset-7/root/usr/lib64:/opt/rh/devtoolset-7/root/usr/lib:/opt/fluendo-codecs-linux-x86_64/dist/linux_x86_64/lib:/opt/fluendo-codecs-linux-x86_64/build-tools/lib:/opt/rh/devtoolset-7/root/usr/lib" ldd -r /usr/lib64/python2.7/site-packages/pycurl.so
        linux-vdso.so.1 =>  (0x00007ffe34fe4000)
        libcurl.so.4 => /opt/fluendo-codecs-linux-x86_64/dist/linux_x86_64/lib/libcurl.so.4 (0x00007f7e4379b000)
        libpython2.7.so.1.0 => /lib64/libpython2.7.so.1.0 (0x00007f7e433cf000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f7e431b3000)
        libc.so.6 => /lib64/libc.so.6 (0x00007f7e42de5000)
        libssl.so.1.1 => /opt/fluendo-codecs-linux-x86_64/dist/linux_x86_64/lib/libssl.so.1.1 (0x00007f7e42b58000)
        libcrypto.so.1.1 => /opt/fluendo-codecs-linux-x86_64/dist/linux_x86_64/lib/libcrypto.so.1.1 (0x00007f7e426a0000)
        libz.so.1 => /opt/fluendo-codecs-linux-x86_64/dist/linux_x86_64/lib/libz.so.1 (0x00007f7e42489000)
        libdl.so.2 => /lib64/libdl.so.2 (0x00007f7e42285000)
        libutil.so.1 => /lib64/libutil.so.1 (0x00007f7e42082000)
        libm.so.6 => /lib64/libm.so.6 (0x00007f7e41d80000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f7e43c08000)
undefined symbol: CRYPTO_num_locks      (/usr/lib64/python2.7/site-packages/pycurl.so)
undefined symbol: CRYPTO_set_id_callback        (/usr/lib64/python2.7/site-packages/pycurl.so)
undefined symbol: CRYPTO_set_locking_callback   (/usr/lib64/python2.7/site-packages/pycurl.so)```